### PR TITLE
Update markets for Berlin (published 18.02.2019).

### PR DESCRIPTION
**Pull request by johnjohndoe**

### CODE REVIEW

File: cities/berlin.json [hunk 1]
```diff
@@ -217,8 +169,8 @@
                 "type": "Point"
             },
             "properties": {
-                "title": "Wochenmarkt Eberbacher Straße",
-                "location": "Wochenmarkt Eberbacher Straße",
+                "title": "Wegen des Feiertages am 08. März 2019 findet dieser Markt bereits am Donnerstag (07.03.) statt",
```
Comment (k-nut) [inline cities/berlin.json]: Do we want to have a `title` like this in our data? Its probably not worth it to add another cleanup step for this, right?

Comment (johnjohndoe) [reply cities/berlin.json]: @k-nut Nice catch. I just mailed the error to the data owner. Seems like we cannot guess the market name meanwhile...?

Comment (johnjohndoe) [reply cities/berlin.json]: I was able to identify the market by its `coordinates`:

- Market at 14.08.2018: [Lines 380-408](https://github.com/wo-ist-markt/wo-ist-markt.github.io/blob/c7668d33d7c909fc14e9bf92c14e5bc0ccd34da1/preprocessing/berlin/raw/markets-berlin.json?short_path=42871be#L380-L408)
- Market at 19.02.2019: [Lines 293-321](https://github.com/wo-ist-markt/wo-ist-markt.github.io/blob/update-markets-berlin/preprocessing/berlin/raw/markets-berlin.json?short_path=8d2cff7#L293-L321)

I fixed the `title` and `location` properties manually.

Comment (johnjohndoe) [reply cities/berlin.json]: Yes, that would make the process much easier to comprehend. I had similar thought the other day. I think the sorting should be applied to the "raw" JSON file which is being downloaded from the data source. Let's take this idea to a new issue.

**Pull request merged**